src/audio/volume.h

```diff
@@ -117,7 +117,7 @@ struct comp_data {
 	/**< volume processing function */
 	void (*scale_vol)(struct comp_dev *dev, struct comp_buffer *sink,
 		struct comp_buffer *source, uint32_t frames);
-	struct work volwork;			/**< volume scheduled work function */
+	struct sof_task volwork;	/**< volume scheduled work function */
```

Comment:
Change field name to e.g. vol_task.
The same for other variables that previously were work.